tuf/signed/verify.go

```diff
@@ -77,10 +77,11 @@ func VerifySignatures(s *data.Signed, roleData data.BaseRole) error {
 		if key.ID() != sig.KeyID {
 			return ErrInvalidKeyID{}
 		}
-		if err := VerifySignature(msg, sig, key); err != nil {
+		if err := VerifySignature(msg, &sig, key); err != nil {
 			logrus.Debugf("continuing b/c %s", err.Error())
 			continue
 		}
+		s.Signatures[i].IsValid = sig.IsValid
```

Comment:
@cyli that sounds good.  The only other place `VerifySignature` is called outside of testing is in `Sign`, and I'm fine with signatures there not having to care about `IsValid` if we aren't going to export the field 👍